Orange/widgets/unsupervised/owdistances.py

```diff
@@ -121,6 +123,8 @@ def commit(self):
             if numpy.product(shape) == 0:
                 self.Error.empty_data(shape)
             else:
+                if bn.anynan(data.X):
```

Comment:
Can this be moved closer to the actual check for missing data and imputation. Reusing the same conditional check if possible (currently appears to be coupled with discretization, so might need some exercise).